x-pack/plugins/logstash/public/sections/index.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { HashRouter, Route, Switch, Redirect } from 'react-router-dom';
+import { Observable } from 'rxjs';
+import { first } from 'rxjs/operators';
+
+import { CoreStart } from 'src/core/public';
+import { ManagementAppMountParams } from '../../../../../src/plugins/management/public';
+import {
+  ClusterService,
+  MonitoringService,
+  PipelineService,
+  PipelinesService,
+  UpgradeService,
+  // @ts-ignore
+} from '../services';
+// @ts-ignore
+import { PipelineList } from '../components/pipeline_list';
+import { PipelineEditView } from './pipeline_edit_view';
+// @ts-ignore
+import { Pipeline } from '../models/pipeline';
+import { SecurityPluginSetup } from '../../../security/public';
+// @ts-ignore
+import * as Breadcrumbs from './breadcrumbs';
+
+export const renderApp = async (
+  core: CoreStart,
+  { basePath, element, setBreadcrumbs }: ManagementAppMountParams,
+  license$: Observable<any>,
+  security?: SecurityPluginSetup
+) => {
+  const logstashLicenseService = await license$.pipe(first()).toPromise();
+  const clusterService = new ClusterService(core.http);
+  const monitoringService = new MonitoringService(
+    core.http,
+    // When monitoring is migrated this should be fetched from monitoring's plugin contract
+    core.injectedMetadata.getInjectedVar('monitoringUiEnabled'),
```

Comment:
Not sure how to make sure this doesn't get missed when #62908 is merged. I will coordinate with @igoristic to make sure this doesn't break.